zds/tutorialv2/forms.py

```diff
@@ -635,7 +636,13 @@ class AcceptValidationForm(forms.Form):
         required=False,
         initial=True
     )
-
+    pubdate = forms.DateField(
+        label=_(u'Date de publication (Laissez vide pour maintenant.)'),
```

Comment:
non.

2016-07-15 19:42 GMT+02:00 victor felder <notifications@github.com>:

> In zds/tutorialv2/forms.py
> <https://github.com/zestedesavoir/zds-site/pull/3276#discussion_r71012072>
> :
>
> > @@ -635,7 +636,13 @@ class AcceptValidationForm(forms.Form):
> >          required=False,
> >          initial=True
> >      )
> > -
> > +    pubdate = forms.DateField(
> > +        label=_(u'Date de publication (Laissez vide pour maintenant.)'),
>
> "pour le moment" ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3276/files/b5e0edf701ffee84f83c42cdb8bb8673cecd8e9e#r71012072>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABc_xeeleRoxmshpqoayKOviNQ_izLA6ks5qV8aXgaJpZM4G_EBV>
> .
>